js/screenswitch.js

```diff
@@ -0,0 +1,20 @@
+var redis = require('redis');
+
+var ScreenSwitchHelper = function(){
```

Comment:
I think if we want to contribute this back to MagicMirror, Redis might be to specific a technology to invoke this mechanism. Can we maybe add another HTTP endpoint to the server that will trigger screen changes?